chapters/functions.tex

```diff
@@ -1502,6 +1502,8 @@ \subsection{Partial Derivatives of Functions}\label{partial-derivatives-of-funct
   h := Gibbs(p, T) - T * Gibbs_T(p, T);
 end specificEnthalpy;
 \end{lstlisting}
+Thus \lstinline!der(Gibbs, T)! corresponds to $\frac{\partial \text{\lstinline!Gibbs!}}{\partial \text{\lstinline!T!}}=\frac{\partial}{\partial \text{\lstinline!T!}}\text{\lstinline!Gibbs!}$.
```

Comment:
When `Gibbs` is in the numerator, shouldn't it appear with the formal parameters written out?

```suggestion
Thus \lstinline!der(Gibbs, T)! corresponds to $\frac{\partial \text{\lstinline!Gibbs(p, T)!}}{\partial \text{\lstinline!T!}} = \frac{\partial}{\partial \text{\lstinline!T!}}\text{\lstinline!Gibbs!}$.
```

Reply:
Sure, it is slightly clearer.